src/move/utils/model_utils.py

```diff
@@ -1,4 +1,5 @@
 import os
+from typing import List, Tuple
```

Comment:
I recently found out that you can have the lastest type annotations using 
`from __future__ import annotations`. This avoids importing the old versions again and again. (So now you can just use `list` and `tuple` as annotations

Reply:
but you only need to do it one in your main `__init__.py` of the package:) ( instead of every python file you want to use it in)